### ISSUE TITLE
the emoji btn can be placed on the left side and the plugin cant be used on a 'ckeditor5 document' textarea

### ISSUE THREAD

**Issue by yar-developer** [mervick/emojionearea#431]
hi, 
1. is there a config for placing the emoji btn on the other(left) side of the input (in the direction:rtl page)? the 'pickerPosition' config doesn't work and it doesn't have 'left' value 
2. i didnt find a config for setting the width of the container? is CSS the only way?
3. is it possible for using it on textarea which already has  'ckeditor 5 document'  installed on it? i tried, but it didn't work

**Comment by yar-developer** [comment]
hi, for number 2, if we want to use CSS, the classes that the plugin produces for two textareas are the same, so it will be a little bit difficult to select, do you have any solution?
and for all the 3 issues above, do you have any planning for development?

**Comment by mervick** [comment]
Yes, I was planning to fix this in a future version, but I cannot say when I will.
Therefore, do not hope for a quick fix.

**Comment by carpii** [comment]
@yar-developer 

If you apply a CSS class to the source INPUT or TEXTAREA, it gets copied over to the resulting EmojiOneArea container div.

You could use this to differentiate..

https://jsfiddle.net/6cjzph3k/
